eo-maven-plugin/src/test/java/org/eolang/maven/ProbeMojoTest.java

```diff
@@ -0,0 +1,187 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2016-2022 Objectionary.com
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.eolang.maven;
+
+import com.yegor256.tojos.MnJson;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.LinkedList;
+import org.cactoos.Input;
+import org.cactoos.io.InputOf;
+import org.cactoos.io.ResourceOf;
+import org.cactoos.text.TextOf;
+import org.eolang.maven.hash.ChCached;
+import org.eolang.maven.hash.ChRemote;
+import org.eolang.maven.objectionary.Objectionary;
+import org.eolang.maven.objectionary.OyRemote;
+import org.eolang.maven.util.Home;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.Matchers;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.api.io.TempDir;
+
+/**
+ * Test case for {@link ProbeMojo}.
+ *
+ * @since 0.28.11
+ */
+@SuppressWarnings("PMD.AvoidDuplicateLiterals")
+final class ProbeMojoTest {
```

Comment:
@MikhailLipanin We are using a new method for testing `Mojos` with using `FakeMaven` class.
The example of the test - [TranspileMojoTest.java](https://github.com/objectionary/eo/blob/master/eo-maven-plugin/src/test/java/org/eolang/maven/TranspileMojoTest.java)
The discussion about the method: https://github.com/objectionary/eo/issues/1417

Reply:
added todo